torch/_inductor/codecache.py

```diff
@@ -346,6 +351,8 @@ def get_hash(content: Union[str, bytes], extra: str = "", hash_type: str = "code
         return code_hash(content, extra)
     if hash_type == "cubin":
         return code_hash(repr(content))
+    if hash_type == "cg":
```

Comment:
mind explaining this ?

Reply:
@eellison it's taken from @ani300's original diff. There are some existing utilities in this file to read/write the entries on disk (see `get_path`, `get_hash`, `write`). I think it basically boils down to an attempt to make the `write` method sorta generic for different cache types in that it computes the hash differently based on the type, supplied as this hash_type arg. It doesn't work all that cleanly here. I can try to improve, but honestly I think the FxGraphCache is going to need its own separate read/write logic anyway in a follow-up PR because your proposal requires that we handle multiple entries per key.